<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStackWindow</class>
 <widget class="QDialog" name="CStackWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stack</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pop selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHide">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPurge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Purge all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectSign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sel for signing</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStaticSign">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Signing Certificate :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignCert">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;none&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
